.github/workflows/publish-environment.yml

```
@@ -23,7 +23,7 @@ jobs:
         uses: MetaMask/action-checkout-and-setup@v1
         with:
           is-high-risk-environment: true
-      - run: yarn build:lavamoat
+      - run: yarn build
```

This ensures we don't build examples, which are unnecessary in this step.
```suggestion
      - name: Build dependencies
        run: yarn build:ci
      - name: Build execution environments
        run: yarn build:lavamoat
```